packages/contracts-bedrock/src/deploy-utils.ts

```diff
@@ -69,38 +84,14 @@ export const deployAndVerifyAndThen = async ({
   await hre.ethers.provider.waitForTransaction(result.transactionHash)
 
   if (result.newlyDeployed) {
-    if (!(await isHardhatNode(hre)) && hre.network.config.live !== false) {
```

Comment:
Removing programmatic verification for now, because of the way the deployment works we can easily just verify after the fact. Breaks currently because we're using foundry under the hood.

Reply:
there is a hh task that does the foundry verification with hh deploy